controller/backup_target_controller.go

```diff
@@ -174,6 +175,24 @@ func getLoggerForBackupTarget(logger logrus.FieldLogger, backupTarget *longhorn.
 	)
 }
 
+func getBackupTargetClient(ds *datastore.DataStore, defaultEngineImage string, backupTarget *longhorn.BackupTarget) (*engineapi.BackupTargetClient, error) {
```

Comment:
This function doesn't need input `defaultEngineImage`. This setting won't change frequently. You can directly fetch this info via `ds`.